test/configuration/shared.config.js

```diff
@@ -20,6 +20,9 @@ exports.testType = {
     ...(TEST_TYPE === 'visual' ? {
         name: 'visual',
         services: ['chromedriver', 'percy'],
-        specs: ['test/visual/*.visual.*.spec.js']
+        specs: [
+            'test/visual/*.visual.*.spec.js',
```

Comment:
If we don't have mobile or desktop ver, do we need this line?

Reply:
we need both for now because some components still use the old file structure - but we'll be refactoring the rest soon so hopefully will be able to get rid of it 🤞

Reply:
Oh of course. That makes sense. 👍🏼